scss/components/_cc-ma-menu.scss

```diff
@@ -100,4 +101,13 @@
     right: 14px;
     font-size: 16px;
   }
+}
+.oo-ma-btn {
```

Comment:
Does ma stand for multi audio? Can we write out 'multi-audio' for this class? It would be more descriptive and have less chance at possible conflicts/confusion with new classes in the future.

Reply:
\+